src/components/Pagination/docs/index.story.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { Pagination } from '../';
+
+export default {
+  category: 'Components',
+  storyName: 'Pagination',
+  component: Pagination,
+  componentPath: '../Pagination.tsx',
+
+  componentProps: setState => ({
+    'data-hook': 'storybook-Pagination',
+    totalPages: 10,
+    maxPagesToShow: 5,
+    currentPage: 1,
+    onChange: ({ page }) => setState({ currentPage: page }),
+  }),
+
+  exampleProps: {
+    alignment: ['left', 'center', 'right'],
+    paginationMode: ['pages', 'input'],
+    onChange: ({ page }) => `Selected page: ${page}`,
+  },
+};
```

Comment:
Please add sections to the story.
You can see an example for it here:
https://github.com/wix/wix-ui-tpa/blob/master/src/components/LikeButton/docs/index.story.tsx

It will gives us a very readable story.